processing/src/main/java/io/druid/segment/IndexMerger.java

```diff
@@ -549,10 +553,10 @@ private File makeIndexFiles(
     IOPeon ioPeon = new TmpFileIOPeon();
     ArrayList<FileOutputSupplier> dimOuts = Lists.newArrayListWithCapacity(mergedDimensions.size());
     Map<String, Integer> dimensionCardinalities = Maps.newHashMap();
-    ArrayList<Map<String, IntBuffer>> dimConversions = Lists.newArrayListWithCapacity(indexes.size());
+    ArrayList<Map<String, int[]>> dimConversions = Lists.newArrayListWithCapacity(indexes.size());
```

Comment:
With direct IntBuffer, it took 8 seconds, not making any notable differences in performance. I prefer int[] but I'm ok with direct buffer. Opinion?